content/en/platform/corda/5.1/deploying-operating/observability/metrics/task-manager.md

```diff
@@ -0,0 +1,55 @@
+| `corda_taskmanager_executor_idle_seconds_count`           | Counter | `name`                                                    | The number of events that have been observed for the base metric.                   |
+| `corda_taskmanager_executor_idle_seconds_max`             | Gauge   | `name`                                                    |                                                                                        |
+| `corda_taskmanager_executor_idle_seconds_sum`             | Counter | `name`                                                    | The total sum of all observed values for the base metric.                           |
+| `corda_taskmanager_executor_pool_core_threads`            | Gauge   | `name`                                                    | The core number of threads for the pool.                                               |
+| `corda_taskmanager_executor_pool_max_threads`             | Gauge   | `name`                                                    | The maximum allowed number of threads in the pool.                                     |
+| `corda_taskmanager_executor_pool_size_threads`            | Gauge   | `name`                                                    | The current number of threads in the pool.                                             |
+| `corda_taskmanager_executor_queue_remaining_tasks`        | Gauge   | `name`                                                    | The number of additional elements that this queue can ideally accept without blocking. |
+| `corda_taskmanager_executor_queued_tasks`                 | Gauge   | `name`                                                    | The approximate number of tasks that are queued for execution.                         |
+| `corda_taskmanager_executor_scheduled_repetitively_total` | Counter | `name`                                                    |                                                                                        |
+| `corda_taskmanager_executor_seconds_count`                | Counter | `name`                                                    | The number of events that have been observed for the base metric.                   |
+| `corda_taskmanager_executor_seconds_max`                  | Gauge   | `name`                                                    |                                                                                        |
+| `corda_taskmanager_executor_seconds_sum`                  | Counter | `name`                                                    | The total sum of all observed values for the base metric                            |
+| `TaskCompletionTime`                                      |         | <ul><li>`task.manager.name`</li><li>`task.type`</li></ul> | The time it took to execute a task, including time waiting to be scheduled.            |
```

Comment:
@lankydan what type is this one?

Reply:
Probably better to form it in the same way as the others:

- `corda_taskmanager_completion_time_count` - Counter - The number of tasks completed by a task manager.
- `corda_taskmanager_completion_time_max` - Gauge - The max time taken to complete a task by a task manager.
- `corda_taskmanager_completion_time_sum` - Counter - The total sum of all completed tasks by a task manager.